packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/LightCurveViewer/LightCurveViewer.js

```diff
@@ -128,8 +125,16 @@ class LightCurveViewer extends Component {
   data points.
   Called when new data (points) is received, and when chart is resized.
    */
-  drawChart (width, height, shouldAnimate = true) {
-    const props = this.props
+  drawChart (shouldAnimate = true) {
+    const {
+      chartStyle,
+      dataExtent,
+      dataPoints,
+      innerMargin,
+      outerMargin
+    } = this.props
+    const container = this.svgContainer.current
+    const { height, width } = container?.getBoundingClientRect()
     if (height && width) {
       this.chartWidth = width
       this.chartHeight = height
```

Comment:
This doesn’t seem to be used anywhere.

Reply:
I agree, `chartWidth` and `chartHeight` appear unused. I introduced them in #385, and these might have been leftovers from an earlier WIP, or possibly some resizing feature that didn't pan out.

There's another part of this code that sets `this.chartWidth = 100` and `this.chartHeight = 100` that's similarly unused.